docs/index.md

```diff
@@ -28,11 +28,12 @@ resource "logdna_view" "http500" {
 
 ## Pre-requirements and considerations
 Before using Terraform for creating resources in LogDNA, review the following notes:
-- Verify Terraform is [installed](https://learn.hashicorp.com/tutorials/terraform/install-cli). The minimum supported version is 0.12.0 and can be checked by running `terraform version`
-- Have the service key for your Organization available. To obtain the service key for your LogDNA Organization, go to the LogDNA dashboard and navigate to **Settings > Organization > API Keys** or follow this link [here](https://app.logdna.com/manage/api-keys)
-- Be aware that the underlying LogDNA Configuration API has a rate limit of 50 requests at a time; therefore, when using the LogDNA Terraform provider, there is also a limit of 50 resource operations at a time
-- If you do not provide a specific base URL in the provider configuration, the base url defaults to `https://api.logdna.com`
-- Authentication is handled via the `servicekey` parameter and can be set in the provider configuration
+- Verify Terraform is [installed](https://learn.hashicorp.com/tutorials/terraform/install-cli). The minimum supported version is 0.12.0 and can be checked by running `terraform version`.
+- Have the service key for your Organization available. To obtain the service key for your LogDNA Organization, go to the LogDNA dashboard and navigate to **Settings > Organization > API Keys** or follow this link [here](https://app.logdna.com/manage/api-keys).
+- Be aware that the underlying LogDNA Configuration API has a rate limit of 50 requests at a time; therefore, when using the LogDNA Terraform provider, there is also a limit of 50 resource operations at a time.
+- If you do not provide a specific base URL in the provider configuration, the base url defaults to `https://api.logdna.com`.
+- Authentication is handled via the `servicekey` parameter and can be set in the provider configuration.
+- Note that if you create a new View using Terraform, but then delete the View by using the LogDNA Dashboard UI and not Terraform, then Terraform will not be aware of the deletion and displays an Error: Resource Not Found message. Should this occur, use terraform state rm on the deleted View resource such that the UI and Terraform are in sync.
 
```

Comment:
One more for the list, please!

- If you want to create an Alert that uses PagerDuty to notify you, you will need to provide LogDNA with the [PagerDuty API key](https://support.pagerduty.com/docs/generating-api-keys). 

Also, can you please move the "Authentication is handled..." bullet point up to be the very first one? I worry it is getting lost in the long list. ;-)

Reply:
I edited the link to go to the anchor "#events-api-keys"

Reply:
sounds good!! i'm moving authentication to the third bullet point as i think it makes most sense to go after us explaining what a service key is and how to obtain it